www/pagestyle2.css

```diff
@@ -48,7 +48,11 @@ select {
   color: #fff;
 }
 
-/* 
+.home a {
```

Comment:
this will nuke those bottom links in the white boxes
![Screen Shot 2022-07-26 at 1 07 17 PM](https://user-images.githubusercontent.com/607541/181102603-d74f3f8d-6033-4437-bea2-187d0f5766ec.png)

Reply:
Good catch, thanks! I didn't see these in my local setup. 

Do you think these should be the exception (and keep the general "all links are white on the homepage"), since all the rest of the page is dark background? Or I should drop the generic `.home a` and fix the offending links in the other parts?

Reply:
I looked around and I couldn't find anything else weird. just those boxes which you can fix by applying a style to the box links